src/components/character-controller.js

```diff
@@ -116,15 +116,27 @@ AFRAME.registerComponent("character-controller", {
   },
 
   onTranslateX: function(event) {
-    this.accelerationInput.setX(event.detail);
+    if (typeof event.detail !== "object") {
+      this.accelerationInput.setX(event.detail);
+    } else {
+      this.accelerationInput.setX(0);
+    }
```

Comment:
Is this a bug in input-mapping that we should submit a PR for? Maybe put a TODO in here.